browser/net/BUILD.gn

```diff
@@ -1,7 +1,10 @@
 import("//build/config/features.gni")
 
 source_set("net") {
-  configs += [ "//brave/build/geolocation" ]
+  configs += [
+    "//brave/build/geolocation",
```

Comment:
I'm not sure that using configs like this is the best way to do this. I think it would be better to create a normal dependency `//brave/browser/safe_browsing` and give it a public config

Reply:
btw - my reasoning for this is that a dep provides more flexibility to manage configs, sources and add any additional deps that might be required in the future. In most cases (there are some exceptions), configs are treated more like internal implementation details for deps and those details are passed along using public_configs